### DISCUSSION TITLE
Connection Error with host.docker.internal in Docker Compose Setup Using External Network

### DISCUSSION THREAD

**Discussion by rutvik2611** [open-webui/open-webui#5103] [General]
# Bug Report

## Installation Method

[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]

## Environment

- **Open WebUI Version:** [e.g., v0.3.11]
- **Ollama (if applicable):** [e.g., v0.2.0, v0.1.32-rc1]
- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior

[Describe what you expected to happen.]

I tried setting the environment variables as follows:

- `OLLAMA_HOST=0.0.0.0`
- `OLLAMA_ORIGINS=*`

However, I'm still encountering a connection error. Below is the `docker-compose.yml` code I am using:

```yaml
version: "3.3"

services:
  open-webui:
    image: ghcr.io/open-webui/open-webui:main
    container_name: open-webui
    restart: always
    # network_mode: host  # Uncomment this line to use host network if needed

    ports:
      - "11434:11434"   # Forward host port 11434 to container port 11434
      - "8088:8080"     # Forward host port 8088 to container port 8080
    volumes:
      - ./persist/open-webui:/app/backend/data
      - /var/run/docker.sock:/var/run/docker.sock # Pass Docker socket into the container
    environment:
      - OLLAMA_BASE_URL=http://127.0.0.1:11434 # Use service name for internal communication
      - OLLAMA_HOST=0.0.0.0
      - OLLAMA_ORIGINS=*

    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.open-webui-http.rule=Host(`open-webui.${MAIN_DOMAIN}`)"
      - "traefik.http.routers.open-webui-http.entrypoints=web"
      - "traefik.http.routers.open-webui-secure.rule=Host(`open-webui.${MAIN_DOMAIN}`)"
      - "traefik.http.routers.open-webui-secure.entrypoints=websecure"
      - "traefik.http.routers.open-webui-secure.tls.certresolver=myresolver"
      - "traefik.http.services.open-webui.loadbalancer.server.port=8080"
    networks:
      backboneNetwork:
        ipv4_address: 172.20.0.11

networks:
  backboneNetwork:
    external: true
    ipam:
      config:
        - subnet: "172.20.0.0/16"
`INFO:     174.49.73.31:57459 - "GET /api/models HTTP/1.1" 200 OK
ERROR [apps.ollama.main] Connection error: Cannot connect to host host.docker.internal:11434 ssl:default [Name or service not known]
INFO:     174.49.73.31:57462 - "GET /api/v1/documents/ HTTP/1.1" 200 OK
`

**Comment by nerdy-tech-com-gitub** [comment]
anyone resolve this yet?

**Reply by nutbolt9** [reply]
docker pull ghcr.io/open-webui/open-webui:latest
